README.md

```diff
@@ -75,6 +75,20 @@ If you encounter an error or have questions, please feel free to file inquiries
 
 Yes! However, in order to make it work, you'll need a "Webhook Relay" that runs from within your network. You can use a Webhook Relay service like Smee (https://smee.io/) or ngrok (https://ngrok.com/) to forward Webhook events to an instance of [Strapi](https://strapi.io/) behind a firewall.
 
+Setup using `smee`:
+
+- After installing `smee` globally run the following command:
+
+  ```
+  smee -u https://smee.io/{CUSTOM_UNIQUE_NAME} -t http://localhost:1337/mux-video-uploader/webhook-handler
+  ```
+
+- Go to Mux **dashboard** -> **settings** -> **webhooks**
+- Click "Create new webhook" and paste `https://smee.io/{CUSTOM_UNIQUE_NAME}`
```

Comment:
We can shortcut this by referencing the Webhooks section.  For example:

- Go to the Webhooks section in the [Mux Dashboard](https://dashboard.mux.com/settings/webhooks) (See the [Webhooks](#Webhooks) section for more info) and use the `https://smee.io/{CUSTOM_UNIQUE_NAME}` as the "URL to notify"